packages/components/nodes/documentloaders/ApifyWebsiteContentCrawler/ApifyWebsiteContentCrawler.ts

```diff
@@ -0,0 +1,68 @@
+import { INode, INodeData, INodeParams } from '../../../src/Interface'
+import { TextSplitter } from 'langchain/text_splitter'
+import { ApifyDatasetLoader } from 'langchain/document_loaders/web/apify_dataset'
+import { Document } from 'langchain/document'
+
+class ApifyWebsiteContentCrawler_DocumentLoaders implements INode {
+    label: string
+    name: string
+    description: string
+    type: string
+    icon: string
+    category: string
+    baseClasses: string[]
+    inputs: INodeParams[]
+
+    constructor() {
+        this.label = 'Apify Website Content Crawler'
+        this.name = 'apifyWebsiteContentCrawler'
+        this.type = 'Document'
+        this.icon = 'apify-symbol-transparent.svg'
+        this.category = 'Document Loaders'
+        this.description = 'Load data from Apify Website Content Crawler'
+        this.baseClasses = [this.type]
+        this.inputs = [
+            {
+                label: 'Apify API Token',
+                name: 'apifyApiToken',
+                type: 'password'
+            },
+            {
+                label: 'Input',
+                name: 'input',
+                type: 'json',
+                default: JSON.stringify({
```

Comment:
Instead of having a json that user has to fill in, can we make it more intuitive to like this? 
```
{
	label: 'URL',
	name: 'url',
	type: 'string',
	placeholder: 'https://js.langchain.com/docs/'
},
{
	label: 'Max Crawl Pages',
	name: 'maxCrawlPages',
	type: 'number',
	default: 1
},
```

Reply:
we can then change the `init` function, get the user input and put into `fromActorCall`